cli/utils.go

```diff
@@ -71,7 +71,7 @@ func getDistroDetails() (name, version string, err error) {
 			}
 		}
 
-		if name != "" && version != "" {
+		if name != "" || version != "" {
```

Comment:
I agree, if we found one of them we should probably return it.
I suspect it might be good if we set the unfound item to something like `<not set>` - and maybe do that for both if neither is found, and not error out at all. @jodh-intel - wdyt?

Reply:
we return what we know and what we don't know(set to <not set> or <unknown> looks better, I think. :)